os/net/mac/tsch/tsch.c

```diff
@@ -928,7 +928,7 @@ PROCESS_THREAD(tsch_send_eb_process, ev, data)
         }
       }
     }
-    if(tsch_current_eb_period > 0) {
+    if(0 && tsch_current_eb_period > 0) {
```

Comment:
you turn off EB here. How do you compensate it? don't see in other code, how EB sends.